apps/web/netlify.toml

```diff
@@ -8,10 +8,29 @@
 [context.deploy-preview]
   command = "cd apps/web && pnpm run envsetup && cd ../../ && pnpm run build:web --skip-nx-cache"
 
+[[redirects]]
+  from = "https://dev.web.novu.co/*"
+  to = "https://dev.dashboard.novu.co/:splat"
```

Comment:
See Netlify [Splat docs](https://docs.netlify.com/routing/redirects/redirect-options/#splats) for more info.

The asterisk `*` converts to `:splat` in their syntax, such that the path is maintained during redirects.